Cython/Compiler/PyrexTypes.py

```diff
@@ -334,6 +337,29 @@ def _assign_from_py_code(self, source_code, result_code, error_pos, code,
             convert_call,
             code.error_goto_if(error_condition or self.error_condition(result_code), error_pos))
 
+    def _generate_xxxref_placeholder(self, *ignored_args, **ignored_kwds):
+        if self.needs_xxxref:
+            raise NotImplementedError("Ref-counting operation not yet implemented for type %s" %
+                                      self)
+        else:
+            return None
```

Comment:
Erm, no. I don't want `None` to accidentally appear in the generated source code anywhere. Maybe an empty string instead?
```suggestion
        return ''
```